### PR TITLE
[ENH] - Add conversion functions for aperiodic measures

### PR DESCRIPTION
**Pull request by TomDonoghue**
This PR add conversion functions between different related aperiodic measures.

### CODE REVIEW

File: neurodsp/aperiodic/conversions.py [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+           Hoyos-Reyes, L. F. (2013). Fractal dimension algorithms and their application
+           to time series associated with natural phenomena. Journal of Physics: Conference
+           Series, 475, 012002. https://doi.org/10.1088/1742-6596/475/1/012002
+
+    Examples
+    --------
+    Convert exponent value to the expected Higuchi fractal dimension:
+
+    >>> convert_exponent_hfd(-1)
+    2.0
```
Comment (ryanhammonds) [inline neurodsp/aperiodic/conversions.py:204]: Under the notes header, it says if exponent = 0, hfd = 2. And then in this example when exponent = -1, hfd = 2.

**Review by ryanhammonds — COMMENTED** [review]
Looks good!

File: neurodsp/aperiodic/conversions.py (hunk 1 above)
Comment (ryanhammonds) [reply neurodsp/aperiodic/conversions.py:204]: Oops, maybe both are correct. So 0 hfd == -1 hfd == 0 exponent?

**Comment by TomDonoghue** [comment]
Since I don't think there's anything more to do about the quirks of Fractal Dimension conversion here, I'm going to merge this in now - but feel free to tag anything else if you to follow up on anything here!

**Pull request merged**